bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/CheckboxesPage.java

```diff
@@ -51,10 +52,10 @@ private void addCheckboxPicker() {
 
         BootstrapCheckBoxPickerConfig config = new BootstrapCheckBoxPickerConfig();
         config
-            .withOnClass("btn-info").withOffClass("btn-warning")
-            .withOnIcon(FontAwesome5IconType.thumbs_up_s).withOffIcon(FontAwesome5IconType.thumbs_down_s)
+            .withOnCls(Type.Info).withOffCls(Type.Warning)
+            .withOnIconCls(FontAwesome5IconType.thumbs_up_s).withOffIconCls(FontAwesome5IconType.thumbs_down_s)
             .withReverse(true)
-            .withStyle(ButtonGroup.Size.Small);
+            .withGroupCls(ButtonGroup.Size.Small);
```

Comment:
Could we preserve the old `with**` methods as `Deprecated` for while ? 
If they can delegate to the new ones it would be awesome! Otherwise they could throw an exception explaining what replaces them.

Reply:
@martin-g sure

Reply:
@martin-g Done!